kinto_client/__init__.py

```diff
@@ -126,6 +133,26 @@ def _get_endpoint(self, name, bucket=None, collection=None, id=None):
         }
         return self.endpoints.get(name, **kwargs)
 
+    def _paginated(self, endpoint, records=None, visited=None):
+        print("paginated")
+        if records is None:
+            records = collections.OrderedDict()
+        if visited is None:
+            visited = set()
+
+        record_resp, headers = self.session.request('get', endpoint)
+        records.update({r['id']: r for r in record_resp['data']})
+        visited.add(endpoint)
+
+        if 'next-page' in map(str.lower, headers.keys()):
+            # Paginated wants a relative URL, but the returned one is absolute.
+            next_page = headers['Next-Page']
+            # Due to bug mozilla-services/cliquet#366, check for recursion:
+            if next_page not in visited:
```

Comment:
We should remove this because @leplatrem handled it with https://github.com/mozilla-services/cliquet/pull/577